examples/srlinux/2node-srl-ixr6-with-oc-services.pbtxt

```diff
@@ -0,0 +1,62 @@
+# This topology consists of two Nokia SR Linux IXR-6e nodes running 22.6.3 release and connected with a single link over ethernet-1/1 interfaces.
+# Each of the nodes comes up with a startup config which is sourced from 22.6.3-opeconfig.cfg.json file;
+# the startup config contains a self-signed TLS cert and has gnmi, gnoi, gribi, p4rt services enabled
+#
+# Read more about this topology at http://learn.srlinux.dev/tutorials/infrastructure/kne/srl-with-kne/
+
+name: "2-srl-ixr6"
+nodes: {
+    name: "srl1"
+    vendor: NOKIA
+    model: "ixr6e"
+    config:{
+        image: "ghcr.io/nokia/srlinux:22.6.3"
+        file: "22.6.3-opeconfig.cfg.json"
+    }
+    services:{
```

Comment:
I read the instructions doc and it seems the other services (gnoi, gribi, p4rt) are also enabled by the config even though they are not specified here.

Ondatra reads the services section of the proto to determine which services are enabled and which ports they are available on. Additionally Ondatra determines the certificate type (self signed, etc.) for connecting to these services from the proto. Since these fields are not defined here we would need to do some post processing on the protobuf to add all of the exposed services and the cert type.

Just so I understand correctly some services (gRIBI, p4rt, etc.) are defined fully by the device config and cannot be specified here?

Reply:
sorry, I missed that. Will add all enabled services.

Reply:
added in [6a08c8c](https://github.com/openconfig/kne/pull/228/commits/6a08c8ccc9d208b446be5529f3c17b790b4bff35)